pandas/core/indexers.py

```diff
@@ -65,18 +65,20 @@ def is_list_like_indexer(key) -> bool:
     return is_list_like(key) and not (isinstance(key, tuple) and type(key) is not tuple)
 
 
-def is_scalar_indexer(indexer, arr_value) -> bool:
+def is_scalar_indexer(indexer, ndim: int) -> bool:
```

Comment:
can you add a parameters section to the docstring while in the neighbourhood and making changes to the parameters